tests/conftest.py

```diff
@@ -1,15 +1,52 @@
 from collections.abc import Generator
+from multiprocessing import Process
+from time import sleep
+from typing import Any
 
 import pytest
+import uvicorn
 from fastapi.testclient import TestClient
-from sqlmodel import Session, SQLModel
+from playwright.sync_api import sync_playwright
+from sqlmodel import Session
 from tad.core.config import settings
 from tad.core.db import get_engine
 from tad.main import app
-from tad.repositories.statuses import StatusesRepository
-from tad.repositories.tasks import TasksRepository
-from tad.services.statuses import StatusesService
-from tad.services.tasks import TasksService
+
+
+def run_server():
+    settings.APP_DATABASE_FILE = "database.sqlite3.test"
+    uvicorn.run(app, host="127.0.0.1", port=8000)
```

Comment:
We should probably pull url to a constant?

If we set port to 0, I believe it will pick an available port. But then we'd have to figure out how to obtain that port?

Reply:
I have moved the host and port to the settings files, this seems the most logical place (for now).